seqr/views/apis/dataset_api_2_3_tests.py

```diff
@@ -126,7 +132,7 @@ def test_add_variants_dataset(self, mock_es_search, mock_get_index_metadata, moc
         mock_es_search.return_value.params.return_value.execute.return_value.aggregations.sample_ids.buckets = [
             {'key': 'NA19675'}, {'key': 'NA19679'}, {'key': 'NA19678_1'},
         ]
-        mock_file_iter.return_value = ['NA19678_1,NA19678']
+        mock_file_iter.return_value = BytesIO('NA19678_1,NA19678\n'.encode('utf-8'))
```

Comment:
Use `BytesIO` here to make the `mock_file_iter` returns the same data type as the `file_iter()`.